pkgs/games/hmcl-bin/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, stdenv
+, file
+, fetchurl
+, makeWrapper
+, jdk
+, alsaLib
```

Comment:
Can we clean up unused args? (btw alsaLib has been renamed to alsa-lib)

Reply:
Fixed, thanks.